src/platform_impl/macos/appkit/event.rs

```diff
@@ -123,6 +152,15 @@ extern_methods!(
         #[sel(stage)]
         pub fn stage(&self) -> NSInteger;
 
+        #[sel(isARepeat)]
+        pub fn is_a_repeat(&self) -> bool;
+
+        #[sel(windowNumber)]
+        pub fn window_number(&self) -> NSInteger;
+
+        #[sel(timestamp)]
+        pub fn timestamp(&self) -> f64;
```

Comment:
Nit: Should return `NSTimeInterval`
```suggestion
        pub fn timestamp(&self) -> NSTimeInterval;
```